lib/management/spnego_krb5.c

```diff
@@ -82,7 +82,7 @@ static int parse_service_full_name(char *service_full_name,
 			char **service_name,
 			char **host_name)
 {
-	char *name, *delim;
+	char *name = NULL, *delim;
```

Comment:
I can not understand why name should be initialized with NULL here.

Reply:
I tested again. -fanalyzer is not finding an issue anymore. Removed.